File: TOMIMSV4/Entity/Transacciones/PreFactura/Trans_Prefactura_Erp/clsBeTrans_prefactura_Erp_det.vb
Public Class clsBeTrans_prefactura_erp_det

    Implements ICloneable

    Public Property corre_cbdetacuerdosservicios As Integer = 0
    Public Property codigoproducto As String = ""
    Public Property dias As Integer = 0
    Public Property monto As Double = 0.00

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Prefactura/clsBeTrans_prefactura_erp_det.cs
namespace WMS.EntityCore.Prefactura
{
    public class clsBeTrans_prefactura_erp_det : ICloneable
    {
        public int corre_cbdetacuerdosservicios { get; set; } = 0;
        public string codigoproducto { get; set; } = "";
        public int dias { get; set; } = 0;
        public double monto { get; set; } = 0.00;

        public clsBeTrans_prefactura_erp_det()
        {
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}